test/e2e/csc_tests.go

```diff
@@ -0,0 +1,133 @@
+package e2e
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	operator "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	"github.com/operator-framework/operator-sdk/pkg/test"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// pollInterval is used by the pollUntilTrue function to define the frequency that
+	// the provided function is ran
```

Comment:
Not sure what you are trying to say. `function is run at`? Moreover did we make these constants so that any function that wants to poll can use it? If yes, please make the comment generic. Or is it specific to `pollUntilTrue()`? If yes, then you should move them into that function.